jupyterlab_hdf/util.py

```diff
@@ -18,11 +20,23 @@ def dsetChunk(dset, row, col):
     return dset[slice(*row), slice(*col)].tolist()
 
 
+def serialize_value(v):
+    """
+    Turns a value into a JSON serializable version
+    """
+    if isinstance(v, list):
+        return [serialize_value(i) for i in v]
+    if isinstance(v, np.ndarray):
+        return serialize_value(v.tolist())
+    if isinstance(v, bytes):
+        return v.decode()
```

Comment:
Can you please explain a little why this `bytes` handling is needed?

Reply:
Bytes cannot be serialized to Jason in python.